Sources/OktaLogger/FileLoggers/FileLoggerDelegate.swift

```diff
@@ -10,7 +10,9 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 import Foundation
-
+#if canImport(LoggerCore)
```

Comment:
```
#if SWIFT_PACKAGE
```
Is probably more safe and less confusing

Reply:
neat! didn't know about it. updated.